docs/VirtueMart.md

```diff
@@ -116,9 +118,122 @@ Everything is ready to go now. Do a small test purchase and make sure the order
 
 ## Customizing VirtueMart BTCPay payment method settings
 
-todo 
+Your VM BTCPay payment method settings can be found in menu: VirtueMart -> Payment Methods. Click on the payment method of type "btcpayvm" you created.
+
+### Section: BTCPay Server connection settings
+
+This is the most important part of the configuration. The data entered here will connect your VirtueMart shop with your counterparty store configured on BTCPay Server.
+
+**BTCPay Server URL**  
+
+URL to your BTCPay Server instance, including protocol e.g. `https://btcpay.yourdomain.com`. 
+
+**API Key**   
+
+Your BTCPay API Key as mentioned [here](#22-create-an-api-key-and-configure-permissions).
+
+**Store ID**   
+
+The store ID of your BTCPay Server store. Can be found on the store settings page. See 8. [here](#22-create-an-api-key-and-configure-permissions)
+
+**Webhook Secret**   
+
+The wehbook secret which was generated on webhook createion, see [here](#23-create-a-webhook-on-btcpay-server)
+ 
+**Webhook callback URL** 
+
+This field is auto-generated by the plugin and helps you when creating the webhook on BTCPay Server. It contains the needed payment method id and parameters to allow processing of callbacks.
+
+### Section: Order states mapping
+
+You can adjust the mapping of BTCPay Server invoice status to VirtueMart order states. On the left are the invoice states and on the right the order states. The defaults here should be good to go - but if you need, you can overwrite them.
+
+VirtueMart order statuses are explained [here](https://docs.virtuemart.net/manual/configuration-menu/order-statuses.html)
+
+Short summary of BTCPay server invoice statuses:
```

Comment:
Can't we just cross-link https://docs.btcpayserver.org/Invoices/#invoice-statuses seems redundant to repeat this over again and make docs look complicated?

Reply:
True, just was to stupid to find it 🤦‍♂️